src/components/interactive-builder/add-question.modal.tsx

```diff
@@ -19,15 +20,15 @@ import {
   Search,
   Select,
   SelectItem,
-  SelectSkeleton,
   Stack,
   Tag,
   TextInput,
   Tile,
+  SelectSkeleton,
 } from '@carbon/react';
-import { ArrowUpRight } from '@carbon/react/icons';
+import { ArrowUpRight, Add, TrashCan } from '@carbon/react/icons';
 import { showSnackbar, useConfig, useDebounce } from '@openmrs/esm-framework';
-import type { ProgramState, RenderType } from '@openmrs/esm-form-engine-lib';
+import type { ProgramState, RenderType } from '@openmrs/openmrs-form-engine-lib';
```

Comment:
Looks like some changes have gotten reverted to a previous state, could you please update your branch and check again?